docs/design/datacontracts/StackWalk.md

```diff
@@ -293,10 +293,6 @@ HijackFrames carry a IP (ReturnAddress) and a pointer to `HijackArgs`. All platf
 
 TailCallFrames are only used on Windows x86 which is not yet supported in the cDAC and therefore not implemented.
 
-#### HelperMethodFrame
-
-HelperMethodFrames are on the way to being removed. They are not currently supported in the cDAC.
```

Comment:
Does this need a mention of "has been removed" when cDAC is used for older runtimes?